src/yardstick/artifact.py

```diff
@@ -504,7 +504,7 @@ class LabelEntryCollection:
     def __init__(self, entries: List[LabelEntry]):
         self.entries = entries
 
-    @functools.cache
+    @functools.cache  # pylint: disable=method-cache-max-size-none
```

Comment:
I feel like the cache size warning makes more sense if we were writing code than ran on a server or something, so I thought this was fine to suppress, but I'm open to changing it.

Reply:
agreed with the reasoning